source/ai/rl/model/breakout_bbf.py

```
@@ -0,0 +1,212 @@
+import torchvision.transforms as transforms
+import numpy as np
+
+
+class MLP(nn.Module):
+    def __init__(self, in_hiddens=512, med_hiddens=512, out_hiddens=512, layers=1,
+                  act=nn.ReLU(), out_act=nn.Identity(), bias=True):
+        super().__init__()
+
+        modules=[]
+        _hiddens = in_hiddens
+        for l in range(layers):
```

Ruff (0.12.2)

Ambiguous variable name: `l`

(E741)